### PR TITLE
refactor(form): remove disabled form state

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
Assigning to myself to complete while @vladikoff takes some well deserved PTO.

File: app/scripts/views/settings/client_disconnect.js [hunk 1]
```diff
@@ -62,7 +62,6 @@ define(function (require, exports, module) {
     },
 
     onChangeRadioButton() {
```
Comment (shane-tomlinson) [inline app/scripts/views/settings/client_disconnect.js]: Is any of this function needed?

**Comment by shane-tomlinson** [comment]
@mozilla/fxa-devs - r?

File: app/scripts/lib/experiment.js [hunk 2]
```diff
@@ -17,7 +17,6 @@ define(function (require, exports, module) {
    * Experiments that are created on startup in `chooseExperiments`.
    */
   const STARTUP_EXPERIMENTS = {
-    'disabledButtonState': BaseExperiment,
```
Comment (vbudhram) [inline app/scripts/lib/experiment.js]: nit: Remove from [docs](https://github.com/mozilla/fxa-content-server/blob/0b3269ed5f4c4a833e6e9d66793389904db5a24b/docs/query-params.md#options-15)

Comment (shane-tomlinson) [reply app/scripts/lib/experiment.js]: > nit: Remove from docs

Thanks for catching that!

Comment (shane-tomlinson) [reply app/scripts/lib/experiment.js]: I was just looking `control` is still available for `emailFirst`. Maybe that section needs to be cleaned up.

Comment (shane-tomlinson) [reply app/scripts/lib/experiment.js]: I'm getting my PRs confused. yes, I'll remove `disabledButtonState`.

**Pull request merged**